books/free-programming-books-in.md

```diff
@@ -0,0 +1,5 @@
+List of free programming books 
+
+for Ruby
+-The Ruby Programming language
+-Polished Ruby
```

Comment:
Please, read the [CONTRIBUTING guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) with calm.

This request doesn't follow neither structure, I encourage you to take a look at other listings to understand how to are organized. 

This kind of code matches better as [issue](https://github.com/EbookFoundation/free-programming-books/issues) Feel free to open one with your wish, following ththe mentioned rules.